### ISSUE TITLE
Qwen2.5-72B-Instruct int4_wo can't use `--gather_generation_logits` on A100

### ISSUE THREAD

**Issue by tann9949** [NVIDIA/TensorRT-LLM#3277]
### Hardware Specs 
- CPU architecture: x86_64
- CPU/Host memory size: 503GB Memory
- GPU properties
  - GPU name: NVIDIA A100
  - GPU memory: 40GB
- Libraries
  - TensorRT-LLM branch: v0.17 (using docker container nvcr.io/nvidia/tritonserver:25.01-trtllm-python-py3)
- NVIDIA driver version:  550.54.15, CUDA12.4
- OS Ubuntu 24.04

### Information

- [x] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [x] My own task or dataset (give details below)

### Reproduction

1. Convert checkpoint:

```
python /app/examples/quantization/quantize.py \
	    --model_dir $model_dir \
	    --dtype=bfloat16 \
	    --batch_size 8 \
	    --qformat int4_wo \
	    --tp_size 2 \
	    --pp_size 1 \
	    --output_dir $out_dir \
	    --calib_size 512 \
	    --device cuda
```

2. Build model (Ended in error)

```
trtllm-build \
       --checkpoint_dir $out_dir \
       --output_dir $engine_dir \
       --max_batch_size=8 \
       --max_input_len 8192 \
       --max_seq_len 32768  \
       --context_fmha enable \
       --paged_kv_cache enable \
       --remove_input_padding enable \
       --gpt_attention_plugin bfloat16 \
       --use_paged_context_fmha enable \
       --gemm_plugin auto \
       --gather_generation_logits
````

### Expected behavior

I expect the model to build without and error such that I can gather generation logits for further use.

### actual behavior

The `trtllm-build` command fails with the following error

```
[TensorRT-LLM] TensorRT-LLM version: 0.17.0.post1
[04/03/2025-16:11:04] [TRT-LLM] [W] Option --paged_kv_cache is deprecated, use --kv_cache_type=paged/disabled instead.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set bert_attention_plugin to auto.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set gpt_attention_plugin to bfloat16.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set gemm_plugin to auto.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set gemm_swiglu_plugin to None.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set fp8_rowwise_gemm_plugin to None.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set nccl_plugin to auto.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set lora_plugin to None.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set moe_plugin to auto.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set mamba_conv1d_plugin to auto.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set low_latency_gemm_plugin to None.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set low_latency_gemm_swiglu_plugin to None.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set context_fmha to True.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set bert_context_fmha_fp32_acc to False.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set remove_input_padding to True.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set reduce_fusion to False.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set user_buffer to False.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set tokens_per_block to 64.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set use_paged_context_fmha to True.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set use_fp8_context_fmha to False.
[04/03/2025-16:11:04] [TRT-LLM] [I] Set multiple_profiles to False.
1       0x7f723b6c0469 /usr/local/lib/python3.12/dist-packages/tensorrt_llm/libs/libtensorrt_llm.so(+0x741469) [0x7f723b6c0469]
2       0x7f723b8cf1c2 getCublasHandle() + 402
3       0x7f723b887436 tensorrt_llm::common::op::AttentionOp::initialize() + 742
4       0x7f70d7ba7550 tensorrt_llm::plugins::GPTAttentionPlugin* tensorrt_llm::plugins::GPTAttentionPluginCommon::cloneImpl<tensorrt_llm::plugins::GPTAttentionPlugin>() const + 1104
5       0x7f738b80c620 /usr/local/tensorrt/lib/libnvinfer.so.10(+0x17eb620) [0x7f738b80c620]
6       0x7f738b731596 /usr/local/tensorrt/lib/libnvinfer.so.10(+0x1710596) [0x7f738b731596]
7       0x7f73b2f002e2 /usr/local/lib/python3.12/dist-packages/tensorrt/tensorrt.so(+0xf62e2) [0x7f73b2f002e2]
8       0x7f73b2e53ae2 /usr/local/lib/python3.12/dist-packages/tensorrt/tensorrt.so(+0x49ae2) [0x7f73b2e53ae2]
9             0x58238f /usr/bin/python3() [0x58238f]
10            0x549225 _PyObject_MakeTpCall + 117
11            0x5d7909 _PyEval_EvalFrameDefault + 2697
12            0x54ce44 /usr/bin/python3() [0x54ce44]
13            0x54b455 PyObject_Call + 277
14            0x5dba03 _PyEval_EvalFrameDefault + 19331
15            0x54ab3a _PyObject_Call_Prepend + 394
16            0x5a3b08 /usr/bin/python3() [0x5a3b08]
17            0x5492ee _PyObject_MakeTpCall + 318
18            0x5d7909 _PyEval_EvalFrameDefault + 2697
19            0x54ce44 /usr/bin/python3() [0x54ce44]
20            0x54b455 PyObject_Call + 277
21            0x5dba03 _PyEval_EvalFrameDefault + 19331
22            0x54ab3a _PyObject_Call_Prepend + 394
23            0x5a3b08 /usr/bin/python3() [0x5a3b08]
24            0x54b3ac PyObject_Call + 108
25            0x5dba03 _PyEval_EvalFrameDefault + 19331
26            0x54ce44 /usr/bin/python3() [0x54ce44]
27            0x54b455 PyObject_Call + 277
28            0x5dba03 _PyEval_EvalFrameDefault + 19331
29            0x54ce44 /usr/bin/python3() [0x54ce44]
30            0x54b455 PyObject_Call + 277
31            0x5dba03 _PyEval_EvalFrameDefault + 19331
32            0x54ab3a _PyObject_Call_Prepend + 394
33            0x5a3b08 /usr/bin/python3() [0x5a3b08]
34            0x54b3ac PyObject_Call + 108
35            0x5dba03 _PyEval_EvalFrameDefault + 19331
36            0x5d5e4b PyEval_EvalCode + 347
37            0x608fd2 /usr/bin/python3() [0x608fd2]
38            0x6b55e3 /usr/bin/python3() [0x6b55e3]
39            0x6b534a _PyRun_SimpleFileObject + 426
40            0x6b517f _PyRun_AnyFileObject + 79
41            0x6bd1e5 Py_RunMain + 949
42            0x6bcccd Py_BytesMain + 45
43      0x7f742ca0e1ca /usr/lib/x86_64-linux-gnu/libc.so.6(+0x2a1ca) [0x7f742ca0e1ca]
44      0x7f742ca0e28b __libc_start_main + 139
45            0x658425 _start + 37
[455ea92ab8b0:00977] *** Process received signal ***
[455ea92ab8b0:00977] Signal: Aborted (6)
[455ea92ab8b0:00977] Signal code:  (-6)
[455ea92ab8b0:00977] [ 0] /usr/lib/x86_64-linux-gnu/libc.so.6(+0x45320)[0x7f742ca29320]
[455ea92ab8b0:00977] [ 1] /usr/lib/x86_64-linux-gnu/libc.so.6(pthread_kill+0x11c)[0x7f742ca82b1c]
[455ea92ab8b0:00977] [ 2] /usr/lib/x86_64-linux-gnu/libc.so.6(gsignal+0x1e)[0x7f742ca2926e]
[455ea92ab8b0:00977] [ 3] /usr/lib/x86_64-linux-gnu/libc.so.6(abort+0xdf)[0x7f742ca0c8ff]
[455ea92ab8b0:00977] [ 4] /usr/lib/x86_64-linux-gnu/libstdc++.so.6(+0xa5ff5)[0x7f738657aff5]
[455ea92ab8b0:00977] [ 5] /usr/lib/x86_64-linux-gnu/libstdc++.so.6(+0xbb0da)[0x7f73865900da]
[455ea92ab8b0:00977] [ 6] /usr/lib/x86_64-linux-gnu/libstdc++.so.6(__cxa_call_terminate+0x33)[0x7f738657a8e6]
[455ea92ab8b0:00977] [ 7] /usr/lib/x86_64-linux-gnu/libstdc++.so.6(__gxx_personality_v0+0x31a)[0x7f738658f8ba]
[455ea92ab8b0:00977] [ 8] /usr/lib/x86_64-linux-gnu/libgcc_s.so.1(+0x22b06)[0x7f742c70cb06]
[455ea92ab8b0:00977] [ 9] /usr/lib/x86_64-linux-gnu/libgcc_s.so.1(_Unwind_Resume+0x12d)[0x7f742c70d5cd]
[455ea92ab8b0:00977] [10] /usr/local/lib/python3.12/dist-packages/tensorrt_llm/libs/libtensorrt_llm.so(+0x7417ff)[0x7f723b6c07ff]
[455ea92ab8b0:00977] [11] /usr/local/lib/python3.12/dist-packages/tensorrt_llm/libs/libtensorrt_llm.so(_ZN12tensorrt_llm6common2op11AttentionOp10initializeEv+0x2e6)[0x7f723b887436]
[455ea92ab8b0:00977] [12] /usr/local/lib/python3.12/dist-packages/tensorrt_llm/libs/libnvinfer_plugin_tensorrt_llm.so(_ZNK12tensorrt_llm7plugins24GPTAttentionPluginCommon9cloneImplINS0_18GPTAttentionPluginEEEPT_v+0x450)[0x7f70d7ba7550]
[455ea92ab8b0:00977] [13] /usr/local/tensorrt/lib/libnvinfer.so.10(+0x17eb620)[0x7f738b80c620]
[455ea92ab8b0:00977] [14] /usr/local/tensorrt/lib/libnvinfer.so.10(+0x1710596)[0x7f738b731596]
[455ea92ab8b0:00977] [15] /usr/local/lib/python3.12/dist-packages/tensorrt/tensorrt.so(+0xf62e2)[0x7f73b2f002e2]
[455ea92ab8b0:00977] [16] /usr/local/lib/python3.12/dist-packages/tensorrt/tensorrt.so(+0x49ae2)[0x7f73b2e53ae2]
[455ea92ab8b0:00977] [17] /usr/bin/python3[0x58238f]
[455ea92ab8b0:00977] [18] /usr/bin/python3(_PyObject_MakeTpCall+0x75)[0x549225]
[455ea92ab8b0:00977] [19] /usr/bin/python3(_PyEval_EvalFrameDefault+0xa89)[0x5d7909]
[455ea92ab8b0:00977] [20] /usr/bin/python3[0x54ce44]
[455ea92ab8b0:00977] [21] /usr/bin/python3(PyObject_Call+0x115)[0x54b455]
[455ea92ab8b0:00977] [22] /usr/bin/python3(_PyEval_EvalFrameDefault+0x4b83)[0x5dba03]
[455ea92ab8b0:00977] [23] /usr/bin/python3(_PyObject_Call_Prepend+0x18a)[0x54ab3a]
[455ea92ab8b0:00977] [24] /usr/bin/python3[0x5a3b08]
[455ea92ab8b0:00977] [25] /usr/bin/python3(_PyObject_MakeTpCall+0x13e)[0x5492ee]
[455ea92ab8b0:00977] [26] /usr/bin/python3(_PyEval_EvalFrameDefault+0xa89)[0x5d7909]
[455ea92ab8b0:00977] [27] /usr/bin/python3[0x54ce44]
[455ea92ab8b0:00977] [28] /usr/bin/python3(PyObject_Call+0x115)[0x54b455]
[455ea92ab8b0:00977] [29] /usr/bin/python3(_PyEval_EvalFrameDefault+0x4b83)[0x5dba03]
[455ea92ab8b0:00977] *** End of error message ***
Aborted (core dumped)
```

### additional notes

I'm not sure what cause this error whether it's about int4_wo or not. Any help is appreciated.

**Comment by yuxianq** [comment]
Close since TensorRT backend will soon be removed (see https://github.com/NVIDIA/TensorRT-LLM/releases#release-v1.3.0rc20)
